CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.1] - 2022-10-25
```

Comment:
We have automation to take care of releasing. This PR should only contain the changes you want in the app , and a new entry in the changelog file under the `Unreleased` section. Ping me, I can show you how to use the automation!

Reply:
Even though this PR might not be necessary anymore, I have fixed this